beta-src/src/webDiplomacyTheme.ts

```diff
@@ -41,9 +41,26 @@ declare module "@mui/material/styles" {
     Russia: CountryColor;
     Italy: CountryColor;
     Turkey: CountryColor;
+    moveOrderSelected: string;
+    move: string;
+    convoy: string;
+    moveFailed: string;
+    moveSupport: string;
+    holdSupport: string;
+    retreat: string;
```

Comment:
Move these into an `arrow` property or something so that they're not placed directly into the palette... 

Example usage: `theme.palette.arrowColors.holdSupport`